ls1intum/Artemis
# `Development`: Create ESLint-compatible styling for environment.override.ts

### Checklist
#### General



### Motivation and Context


The current `prebuild.mjs` file generates an override file `environment.override.ts` which is incompatible with our ESLint rules:

<img width="696" alt="screenshot-2023-05-25_001385" src="https://github.com/ls1intum/Artemis/assets/9006596/c1f7ee58-a779-4192-9a9a-e68d4270eeff">

We should use single quotation marks `'` to prevent linting errors from showing up in the IDE.

### Description

The prebuild script is changed to use single quotation marks. It now doesn't use `JSON.stringify` but just hardcoded quotation marks instead, which is not a problem: Neither the commit hash nor the version number should ever have any characters in them which need to be encoded.

### Steps for Testing

- Run `npm run prebuild` locally
- Open `environment.override.ts` in your IDE
- Deploy the build to a test server to make sure that the PR didn't break the build (looking at the version number on the top left should be enough)

### Review Progress






### Screenshots



No ESLint errors:
<img width="701" alt="screenshot-2023-05-25_001386" src="https://github.com/ls1intum/Artemis/assets/9006596/08ac4a14-db1e-447d-a1c8-d1806c71333c">
